RELEASE_NOTES.md

```diff
@@ -55,28 +51,37 @@ Produced programs are fully self-sufficient and do not need JVM or other runtime
 
 On macOS it also requires Xcode 11.0 or newer to be installed.
 
-The language and library version supported by this release match Kotlin 1.4.
+The language and library version supported by this release match Kotlin 1.5.
 However, there are certain limitations, see section [Known Limitations](#limitations).
 
  Currently _Kotlin/Native_ uses reference counting based memory management scheme with a cycle
 collection algorithm. Multiple threads could be used, but objects must be explicitly transferred
-between threads, and same object couldn't be accessed by two threads concurrently.
+between threads, and same object couldn't be accessed by two threads concurrently unless it is frozen.
+See the relevant [documentation](https://kotlinlang.org/docs/reference/native/concurrency.html).
+We are going to lift these multithreading restrictions, which involves implementing a new memory manager.
+More details are available in
+["Kotlin/Native Memory Management Roadmap"](https://blog.jetbrains.com/kotlin/2020/07/kotlin-native-memory-management-roadmap/).
 
-_Kotlin/Native_ provides efficient interoperability with libraries written in C or Objective-C, and supports
-automatic generation of Kotlin bindings from a C/Objective-C header file.
-See the samples coming with the distribution.
+_Kotlin/Native_ provides efficient bidirectional interoperability with C and Objective-C.
+See the [samples](https://github.com/JetBrains/kotlin-native/tree/master/samples)
+and the [tutorials](https://kotlinlang.org/docs/tutorials/).
```

Comment:
We'll have to return to doc links once we completely migrate the docs to WebHelp, i.e. next week (we hope 🤞).
We can change this one to https://kotlinlang.org/docs/native-c-interop.html

Reply:
Ok, makes sense. Thanks!